src/pycrunch/elements.py

```diff
@@ -232,7 +232,9 @@ def post(self, data, *args, **kwargs):
         kwargs["headers"].setdefault("Content-Type", "application/json")
         if not isinstance(data, six.string_types):
             data = json.dumps(data)
-        return self.session.post(self.self, data, *args, **kwargs)
+        # Remove any existing params, such as "limit=0"
+        url = self.self.rsplit("?", 1)[0]
```

Comment:
Let's use `urlparse` here instead of split to split the URL.

Reply:
@jjdelc I have replaced `rsplit` with a `urlparse` implementation